src/utils/substrate-rpc.ts

```diff
@@ -0,0 +1,54 @@
+import { Keyring } from '@polkadot/keyring'
```

Comment:
Make this dynamic import

Reply:
Resolved this dynamic import already. I also checked the load size and it's well okay now